### ISSUE TITLE
Jetpack standalone plugins: Guidelines

### ISSUE THREAD

**Issue by crisbusquets** [Automattic/jetpack#37463]
First iteration: p1HpG7-sDj-p2

**Comment by keoshi** [comment]
@crisbusquets @ilonagl Do we need to do anything else with this one?

Not sure if warranted, but would it be helpful to create a plugin guidelines pages in Jetpack P2 for people to easily find it?

**Comment by crisbusquets** [comment]
> but would it be helpful to create a plugin guidelines pages in Jetpack P2 for people to easily find it?

Totally. But I think we should wait until we have clear design guidelines 😅 or at least one standalone plugin we can use as an example.

**Comment by ilonagl** [comment]
yeah, especially if we want to bring everything into one the main plugin.
